suite-native/module-check-backup/src/hooks/useNavigateCheckBackup.ts

```diff
@@ -0,0 +1,40 @@
+import { useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { selectDeviceModel } from '@suite-common/wallet-core';
+import {
+    DeviceCheckBackupStackRoutes,
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { DeviceModelInternal, models } from '@trezor/device-utils';
+
+const checkBackupUnsupportedDeviceModels: Array<DeviceModelInternal> = [DeviceModelInternal.T1B1];
```

Comment:
nit: I think we're using this as a general code style thing, but again, just a nit that can be resolved
```suggestion
const checkBackupUnsupportedDeviceModels: DeviceModelInternal[] = [DeviceModelInternal.T1B1];
```